Core/Core/Quizzes/QuizDetails/View/QuizDetailsView.swift

```diff
@@ -0,0 +1,229 @@
+                }
+                Spacer()
+            }
+                .padding(.top, 2)
+        }
+
+        Divider().padding(.horizontal, 16)
+
+        if let assDateSectionViewModel = viewModel.assignmentDateSectionViewModel {
+            DateSection(viewModel: assDateSectionViewModel)
+            Divider().padding(.horizontal, 16)
+        } else if let quizDateSectionViewModel = viewModel.quizDateSectionViewModel {
+            DateSection(viewModel: quizDateSectionViewModel)
+            Divider().padding(.horizontal, 16)
+        }
+
+        if viewModel.showSubmissions {
+            if let assViewModel = viewModel.assignmentSubmissionBreakdownViewModel {
+                SubmissionBreakdown(viewModel: assViewModel)
+            } else if let quizViewModel = viewModel.quizSubmissionBreakdownViewModel {
+                SubmissionBreakdown(viewModel: quizViewModel)
+            }
+            Divider().padding(.horizontal, 16)
+        }
+
+        if let html = viewModel.quizDetailsHTML, !html.isEmpty {
+            Text("Description", bundle: .core)
+                .font(.medium16).foregroundColor(.textDark)
+                .padding(EdgeInsets(top: 16, leading: 16, bottom: 0, trailing: 16))
+            WebView(html: html)
+                .frameToFit()
+        } else {
+            QuizDetailsSection(label: Text("Description", bundle: .core)) {
+                HStack {
+                    Text("Help your students with this assignment by adding instructions.", bundle: .core)
+                        .font(.regular14).foregroundColor(.textDark)
+                        .fixedSize(horizontal: false, vertical: true)
+                    Spacer()
+                }
+                    .padding(.horizontal, 12).padding(.vertical, 8)
+                    .background(Color.backgroundLight)
+                    .cornerRadius(3)
+                    .padding(.top, 4)
+            }
+        }
+
+        Divider().padding(.horizontal, 16)
+
+        quizAttributes()
+
+        Spacer()
+
+        Button(action: previewQuiz, label: {
+            HStack {
+                Spacer()
+                Text("Preview Quiz", bundle: .core)
+                    .font(.semibold16).foregroundColor(Color(Brand.shared.buttonPrimaryText))
+                Spacer()
+            }
+                .frame(minHeight: 51)
+        })
+            .background(Color(Brand.shared.buttonPrimaryBackground))
+            .cornerRadius(4)
+            .padding(EdgeInsets(top: 0, leading: 16, bottom: 16, trailing: 16))
+    }
+
+    @ViewBuilder
+    func quizAttributes() -> some View {
+        VStack(alignment: .leading, spacing: 4) {
+            let attributes = viewModel.attributes
+            ForEach(attributes) { attribute in
+                HStack(spacing: 4) {
+                    Text(attribute.id).font(.semibold16)
+                    Text(attribute.value)
```

Comment:
Let's group these to one VoiceOver element so we can move from row to row with one swipe.